src/dex-ui/services/HederaService/dao/TokenDAOService.ts

```diff
@@ -145,6 +143,58 @@ async function sendProposeTokenTransferTransaction(params: SendProposeTokenTrans
   return sendProposeTokenTransferTransactionResponse;
 }
 
+interface TokenAssociateTransactionParams {
+  title: string;
+  description: string;
+  linkToDiscussion: string;
+  tokenId: string;
+  governanceTokenId: string;
+  daoAccountId: string;
+  governanceAddress: string;
+  nftTokenSerialId: number;
+  signer: HashConnectSigner;
+}
+
+async function sendGOVTokenAssociateTransaction(params: TokenAssociateTransactionParams) {
+  const {
+    title,
+    description,
+    linkToDiscussion,
+    tokenId,
+    governanceTokenId,
+    daoAccountId,
+    governanceAddress,
+    nftTokenSerialId,
+    signer,
+  } = params;
+  const tokenSolidityAddress = TokenId.fromString(tokenId).toSolidityAddress();
+  const spenderContractId = AccountId.fromSolidityAddress(governanceAddress).toString();
+  const contractFunctionParameters = new ContractFunctionParameters()
+    .addString(title)
+    .addString(description)
+    .addString(linkToDiscussion)
+    .addAddress(tokenSolidityAddress)
+    .addUint256(nftTokenSerialId);
+  await DexService.setTokenAllowance({
+    tokenId: governanceTokenId,
+    walletId: signer.getAccountId().toString(),
+    spenderContractId,
+    tokenAmount: 1 * DEX_PRECISION,
```

Comment:
I believe the decimal value of the governance token should be used to calculate the amount instead of `DEX_PRECISION`.  The DAO logic shouldn't have any dependencies on the DEX code (we will be separating these two into independent applications).

Reply:
Done.